File: android/app/src/main/java/co/searchrestaurant/android/app/fetch/GeocodingApi.java
package co.searchrestaurant.android.app.fetch;

import retrofit2.Call;
import retrofit2.Callback;
import retrofit2.http.GET;
import retrofit2.http.Query;

/**
 * Created by hassanabid on 1/17/16.
 */
public interface GeocodingApi {

    @GET("maps/api/geocode/json")
    Call<GeocodingResponse> getLatLng(@Query("address") String address, @Query("key") String key);


}

File: v2/android/app/src/main/java/dev/hassanabid/searchrestaurant/service/GeoCodingApi.kt
package dev.hassanabid.searchrestaurant.service

import dev.hassanabid.searchrestaurant.service.response.GeocodingResponse
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface GeocodingApi {

    @GET("maps/api/geocode/json")
    suspend fun getLatLng(@Query("address") address: String, @Query("key") key: String): GeocodingResponse


}